packages/auth/samples/saml/index.html

```diff
@@ -0,0 +1,22 @@
+<html>
+  <head>
+    <script type="module">
+      import {buildSamlClient} from '../../src/auth.ts';
+
+      // Test values for "organizationId and "provider" can be generated using:
+      // https://github.com/coveo/searchfuncttests/blob/develop/searchapi/auth/samlOktaMultipleProviderWithQuery.ts
```

Comment:
This comment is probably not enough ... there some setup involved to run that test file including setting up env variables and creating an Okta account.

Maybe I'll create a confluence doc on Friday ... atm the steps are part of the slack lore.
@acote-coveo, @dreisch-coveo do we have a doc by any chance? If not any preferences on where to create one?

Reply:
I think if we need further documentation on how to run those test it should be in the README. I can add more details to it tomorrow

Reply:
Is it really important that people can run this ? What's the value in that ?

Isn't this the important documentation ?  https://docs.coveo.com/en/91/build-a-search-ui/saml-authentication

I think we should also write an article in docs.coveo.com/headless > Usage > SAML

Reply:
Who's the targeted audience for `samples` in Headless? It people outside from Coveo have access to that, I don't see how relevant it is to share a private repo link ^^

Reply:
I had this file in mind more for e2e manual testing vs. a public audience. I think I'll rename `samples` -> `manual-e2e`.

An automated e2e test would be nice too, though there some set up involved.